app/controllers/content_items_controller.rb

```diff
@@ -136,16 +136,41 @@ def set_guide_draft_access_token
     @content_item.draft_access_token = params[:token]
   end
 
+  class GdsApi::Response
+    def update_response_body
+      updated_response = JSON.parse(@http_response.body).dig("data", "edition").deep_transform_keys(&:underscore)
+      @http_response = RestClient::Response.create(
+        updated_response.to_json,
+        @http_response.net_http_res,
+        @http_response.request,
+      )
```

Comment:
**question:** documentation

I'm not sure where this instance variable comes from or what the impact of reassigning it is - is there some documentation anywhere/can this be documented? (I can see what you're trying to do, but I'm struggling to say "yes, it clearly does that")

Reply:
This is being assigned in `GdsApi::Response`'s [initialiser](https://github.com/alphagov/gds-api-adapters/blob/5bbcd887b054126c28a8584baffed2875ee86d3c/lib/gds_api/response.rb#L98).  This code basically just adds a method to that class.

Reply:
I've now moved this out of the application and into gds-api-adapters: https://github.com/alphagov/gds-api-adapters/pull/1310.